src/main/java/solvd/ermakovich/zt/kafka/srteams/config/health/indicator/filter/BloodPressureFilter.java

```diff
@@ -0,0 +1,32 @@
+package solvd.ermakovich.zt.kafka.srteams.config.health.indicator.filter;
+
+import java.util.function.BiPredicate;
+import org.springframework.stereotype.Component;
+import solvd.ermakovich.zt.domain.health.indicators.DefaultHealthIndicators;
+import solvd.ermakovich.zt.domain.health.indicators.HealthIndicatorsMessage;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@Component
+final class BloodPressureFilter implements
```

Comment:
@kermakovich
also, class can be renamed to something like `BloodPressureLessThenCritical`

Reply:
@h1alexbel I think it`s not good, because if `BloodPressureLessThenCritical` returns true, it means that blood pressure is not ok and blood pressure can be also more then critical. So can I name it just as `NormalBloodPressure`?